### PR TITLE
refactor: several ui fixes on demo, banner image and metadata

### PR DESCRIPTION
**Pull request by enesozturk**
# Description

Includes several changes at once:

- Use `ResizeObserver` for connect view's scroll mask image to set mask image style dynamically - when we enable/disable connect options on Demo, even though the page got smaller, we still see scroll mask. Fixing that
- Adds OG images and updates metadata of the Demo app
- Fixes social options broken style when do drag & drop on Demo app
- Padding & margins on Demo app

## Type of change

- [ ] Chore (non-breaking change that addresses non-functional tasks, maintenance, or code quality improvements)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# Associated Issues

For Linear issues: Closes APKT-1729, APKT-1719, APKT-1731, APKT-1782, APKT-1795
For GH issues: closes #...

# Showcase (Optional)

If there is a UI change include the screenshots with before and after state.
If new feature is being introduced, include the link to demo recording.

# Checklist

- [ ] Code in this PR is covered by automated tests (Unit tests, E2E tests)
- [x] My changes generate no new warnings
- [x] I have reviewed my own code
- [ ] I have filled out all required sections
- [ ] I have tested my changes on the preview link
- [ ] Approver of this PR confirms that the changes are tested on the preview link

### CODE REVIEW

File: apps/builder/app/globals.css [hunk 1]
```diff
@@ -124,4 +126,12 @@ body {
   .ease {
     transition-timing-function: cubic-bezier(0.45, 0, 0.55, 1);
   }
+  .page-container {
+    background-image: radial-gradient(
+      hsl(0deg 0% 43.37% / var(--page-background-image-opacity)) 1px,
```
Comment (tomiir) [inline apps/builder/app/globals.css:131]: is this ok? why such a weird value?

File: packages/scaffold-ui/src/views/w3m-connect-view/index.ts [hunk 2]
```diff
@@ -324,23 +329,43 @@ export class W3mConnectView extends LitElement {
...
-    )
-    connectEl.style.setProperty(
-      '--connect-scroll--bottom-opacity',
-      MathUtil.interpolate(
-        [0, 50],
-        [0, 1],
-        connectEl.scrollHeight - connectEl.scrollTop - connectEl.offsetHeight
-      ).toString()
-    )
+    const shouldApplyMask = connectEl.scrollHeight > 470
```
Comment (tomiir) [inline packages/scaffold-ui/src/views/w3m-connect-view/index.ts:336]: should we make this 470 a constant?

File: packages/scaffold-ui/test/views/w3m-connect-view.test.ts [hunk 3]
```diff
@@ -24,6 +24,15 @@ const INSTALLED_WALLET = {
...
 
+// Mock ResizeObserver
+beforeAll(() => {
+  global.ResizeObserver = class {
+    observe() {}
+    unobserve() {}
+    disconnect() {}
+  }
+})
+
```
Comment (tomiir) [inline packages/scaffold-ui/test/views/w3m-connect-view.test.ts:35]: are you doing anything with this on the test?

File: packages/scaffold-ui/src/views/w3m-connect-view/index.ts [hunk 4]
```diff
@@ -49,15 +51,18 @@ export class W3mConnectView extends LitElement {
...
   public override firstUpdated() {
     const connectEl = this.shadowRoot?.querySelector('.connect')
-    // Use requestAnimationFrame to access scroll properties before the next repaint
-    requestAnimationFrame(this.handleConnectListScroll.bind(this))
-    connectEl?.addEventListener('scroll', this.handleConnectListScroll.bind(this))
+    if (connectEl) {
+      this.resizeObserver = new ResizeObserver(() => {
+        this.handleConnectListScroll()
+      })
+      this.resizeObserver.observe(connectEl)
```
Comment (zoruka) [inline packages/scaffold-ui/src/views/w3m-connect-view/index.ts:63]: do we need to unbind this at any point?

Comment (enesozturk) [reply packages/scaffold-ui/src/views/w3m-connect-view/index.ts:68]: `this.resizeObserver?.disconnect()` does the thing in `disconnectedCallback`

File: apps/builder/app/globals.css (hunk 1 above)
Comment (enesozturk) [reply apps/builder/app/globals.css]: Confirming with the designer

Comment (enesozturk) [reply apps/builder/app/globals.css]: Changed & confirmed with the designer

**Pull request merged**